litmuschaos/install.sh

```diff
@@ -1,4 +1,3 @@
 kubectl create ns litmus
 helm repo add litmuschaos https://litmuschaos.github.io/litmus-helm
-helm search repo litmuschaos --devel
-helm install chaos litmuschaos/litmus-2-0-0-beta --devel --namespace=litmus
+helm install chaos litmuschaos/litmus --namespace=litmus
```

Comment:
Can we point to a specific version , this will always get the latest I believe

Reply:
fixed